### ISSUE TITLE
Create sections for intended audience

### ISSUE THREAD

**Issue by bean5** [apache/couchdb-documentation#539]
When I read CouchDB documentation, I'm flooded in by extra verbiage that doesn't need to be there and too much information. It is okay to mention that in reality there are 1600 nodes in a B-tree, but if I understand the need for Big-O of n * Log(n), I probably know that trees can get large and deep.

Is the intended audience someone who is maintaining this source code or someone learning CouchDB?

For a project to grow, it seems more important to be written for the CouchDB beginner than for the maintainer so you can gain traction, especially when there is tight competition.

Frankly, I am worried. I am worried that CouchDB is simply going to lose over time against Firebase not because its technology is worse, but because Firebase's documentation was quick to follow and made it clear that it takes: ~5 lines to set up, ~5 lines to do a write, and ~10 lines to so a search. Meanwhile with CouchDB I'm still busy reading about nodes and realizing that documentation is out of date.

See also: #538

**Comment by wohali** [comment]
Duplicate of #273

**Comment by bean5** [comment]
Mature is one thing, but mature and accepted and growing faster is another. I believe more can be done. I'm trying to understand CouchDB and trying to make the case against Firebase for my projects. Documentation is part of a project and I want to be able to say:

* It is mature
* It is robust
* It is well documented and easy to learn
* It is easy to set up and maintain
* It is worth it
* It is cost effective
* It is secure

CouchDB in the long run would likely be all of this, but the case would be more clear if there was high-level documentation for the code engineer and entirely different section for the DevOps/Maintainers. But that might be personal preference because I come from MySQL/SQLite background that recently waded into PouchDB documenation.

I do not mean to be negative. I think there is great value in the current documentation for DevOps and maintainers. I'm not saying it is garbage. I'm saying for new engineers, some of it is a bit wordy, hard to follow, and seems to cater to 3 audiences at once.

Documentation similar to PouchDB was what I expected. I'm fine if you disagree. I'm just one voice. If I'm the minority, by all means, just ignore. I would do the same. 10 years ago I wouldn't have even made a ticket for this because it was more of a discussion and feedback than "There is a bug here. I have a PR on the way shortly."
